pkg/restapi/operation/didexchange/didexchange.go

```diff
@@ -52,7 +53,7 @@ type provider interface {
 }
 
 // New returns new DID Exchange rest client protocol instance
-func New(ctx provider) (*Operation, error) {
+func New(ctx provider, webhookURLs []string) (*Operation, error) {
```

Comment:
Rather than passing the webhookURLs and constructing [webhookNotifier](https://github.com/hyperledger/aries-framework-go/pull/523/files#diff-5213b774cf71e1cef26c7bb7c48ee94eR79), can we just pass webhookNotifier API directly here ? Individual Protocol's doesn't need to construct webhookNotifier and they can directly use the API ie. `webhookNotifier.Notify(msg)`.

Reply:
Done